### DISCUSSION TITLE
gow3

### DISCUSSION THREAD

**Discussion by hsjsjsjdjdjdhshssh** [bucanero/pkgi-ps3#124] [General]
hello bro can you please add god of war 3 to pkgi pls just this game

**Comment by Masamune3210** [comment]
Please search for existing issues first before making a new one. Pkgj does not have anything to do with what games are or are not on the list. It is just a client for nopaystation. If you want a game added to the list, buy the game and dump the required information that NPS needs so they can add it to the database, until someone does that, the game will not ever be on there.
